server/modules/doppler-client.js

```diff
@@ -0,0 +1,145 @@
+const querystring = require('querystring');
+const fetch = require('node-fetch');
+const shopify = require ('./shopify-extras');
+const baseUrl = 'https://restapi.fromdoppler.com';
+
+const getCustomerFieldValue = function(customer, fieldPath) {
+    let currentProperty = customer;
+    
+    fieldPath.split(".").forEach(propertyName => {
+        currentProperty = currentProperty[propertyName];
+    });
```

Comment:
This looks very clever but also very fragile to edge cases. Do you have a concrete example of inputs?

Reply:
I thought about that, but since it's code handled by us (the propertyName is not be modifiable by the user) I didn't think that it worth to add more complex logic. Anyway, I understand that it's prone to errors. All the possible propertyNames are [these](https://github.com/MakingSense/doppler-for-shopify/pull/3/files#diff-dfd4565b48358215792597dd83a2e5cc) they're the "path" property in each customer field. Let me think if I can find another more robust solution for this.

Reply:
Nah, I think that it is fine by the moment. Thanks!